build.gradle

```diff
@@ -3,21 +3,19 @@ apply plugin: 'com.vanniktech.code.quality.tools'
 
 buildscript {
   ext.versions = [
-    kotlin: '1.3.72',
+    kotlin: '1.4.32',
     minSdk: 9,
     compileSdk: 30,
-    lint: '27.2.1',
+    lint: '30.0.1',
   ]
 
   ext.deps = [
-    'kotlin': [
-      'stdlib': "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${versions.kotlin}",
-    ],
     'lint': [
       'core': "com.android.tools.lint:lint:${versions.lint}",
       'api': "com.android.tools.lint:lint-api:${versions.lint}",
       'tests': "com.android.tools.lint:lint-tests:${versions.lint}",
     ],
+    junit: 'junit:junit:4.13.2',
```

Comment:
Why was junit required to be added? 

My thought was that the kotlin stdlib dependency was removed and maybe contained it but I don't see any mention of junit being included [here](https://kotlinlang.org/api/latest/jvm/stdlib/)

Reply:
IIRC `lint-tests` artifact added this previously but no longer does hits